staging/src/k8s.io/code-generator/cmd/import-boss/README.md

```diff
@@ -0,0 +1,60 @@
+## Purpose
+
+- `import-boss` enforces import restrictions in a given repository.
+
+## How does it work?
+
+- When a directory is verified, `import-boss` looks for a file called `.import-restrictions`. If this file is not found, parent directories will be recursively searched.
+  - NOTE: Adding `.import-restrictions` files does not add them to CI runs. In order for them to be added, it needs to be explicitly added to `arguments.InputDirs`, like so:
```

Comment:
being super verbose but could you specify that `arguments.InoutDirs` is defined in the src code (main.go file in the same dir)